src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Invitations/InvitationsControllerTests.cs

```diff
@@ -11,27 +14,249 @@ public async Task GetInvitation_AsViewer_ShouldGetInvitation()
+                UserType.Viewer,
+                TestFactory.PlantWithAccess,
+                id);
+
+            // Assert
+            Assert.IsTrue(id > 0);
+            Assert.IsNotNull(invitation);
+        }
+
+        [TestMethod]
+        public async Task EditInvitation_AsPlanner_ShouldEditInvitation()
+        {
+            // Arrange
+            var id = await InvitationsControllerTestsHelper.CreateInvitationAsync(
+                UserType.Planner,
+                TestFactory.PlantWithAccess,
+                "InvitationToBeUpdatedTitle",
+                "InvitationToBeUpdatedDescription",
+                "InvitationToBeUpdatedLocation",
+                DisciplineType.DP,
+                _invitationStartTime,
+                _invitationEndTime,
+                _participants,
+                _mcPkgScope,
+                null);
 
-            //// Assert
-            //Assert.IsTrue(id > 0);
-        //}
+            var invitation = await InvitationsControllerTestsHelper.GetInvitationAsync(
+                UserType.Viewer,
+                TestFactory.PlantWithAccess,
+                id);
+
+            invitation.Status = IpoStatus.Planned;
+
+            var currentRowVersion = invitation.RowVersion;
+            const string UpdatedTitle = "UpdatedInvitationTitle";
+            const string UpdatedDescription = "UpdatedInvitationDescription";
+
+            var editInvitationDto = new EditInvitationDto
+            {
+                Title = UpdatedTitle,
+                Description = UpdatedDescription,
+                StartTime = invitation.StartTimeUtc,
+                EndTime = invitation.EndTimeUtc,
+                Location = invitation.Location,
+                ProjectName = invitation.ProjectName,
+                RowVersion = invitation.RowVersion,
+                UpdatedParticipants = ConvertToParticipantDtoEdit(invitation.Participants),
+                UpdatedCommPkgScope = null,
+                UpdatedMcPkgScope = _mcPkgScope
+            };
+
+            // Act
+            var newRowVersion = await InvitationsControllerTestsHelper.EditInvitationAsync(
+                UserType.Planner,
+                TestFactory.PlantWithAccess,
+                id,
+                editInvitationDto);
+
+            var updatedInvitation = await InvitationsControllerTestsHelper.GetInvitationAsync(
+                UserType.Viewer,
+                TestFactory.PlantWithAccess,
+                id);
+
+            // Assert
+            AssertRowVersionChange(currentRowVersion, newRowVersion);
+            Assert.AreEqual(UpdatedTitle, updatedInvitation.Title);
+            Assert.AreEqual(UpdatedDescription, updatedInvitation.Description);
+        }
+
+        [TestMethod]
+        public async Task UploadAttachment_AsPlanner_ShouldUploadAttachment()
+        {
+            // Arrange
+            var invitationIdForAttachment = await InvitationsControllerTestsHelper.CreateInvitationAsync(
```

Comment:
Fine for me, but in this context you could have used existing Invitation found by InitialInvitationId for "attachment container" ... no need for creating a new invit.